From 617c0a64b678e97024649d4d22a2d53c488cd02e Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Mon, 30 Mar 2020 13:50:10 +0700
Subject: [PATCH 145/458] arm64: dts: renesas: r8a774b1: add threshold
 temperature to enable IPA

This patch adds threshold temperature to enable IPA function.
Based on commit <caf18a66833a1172dbe932c9affae0c93155a814> from
horms/renesas-bsp.git repo to apply this similar feature for RZ/G2N.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774b1.dtsi | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a774b1.dtsi b/arch/arm64/boot/dts/renesas/r8a774b1.dtsi
index a5691e7..f9e4c1c 100644
--- a/arch/arm64/boot/dts/renesas/r8a774b1.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a774b1.dtsi
@@ -90,6 +90,8 @@
 			next-level-cache = <&L2_CA57>;
 			enable-method = "psci";
 			#cooling-cells = <2>;
+			cooling-min-level = <0>;
+			cooling-max-level = <2>;
 			dynamic-power-coefficient = <854>;
 			clocks = <&cpg CPG_CORE R8A774B1_CLK_Z>;
 			operating-points-v2 = <&cluster0_opp>;
@@ -2748,6 +2750,13 @@
 				};
 			};
 			trips {
+				threshold: trip-point0 {
+					/* miliCelsius  */
+					temperature = <90000>;
+					hysteresis = <1000>;
+					type = "passive";
+				};
+
 				target: trip-point1 {
 					temperature = <100000>;
 					hysteresis = <1000>;
-- 
2.7.4

